### PR TITLE
Modal - Create Events

### LINKED ISSUE #132: Frontend: Modal View to create a new events
When I am logged in as an admin, I want to be able to create a new event and fill in the info for it.

Create a Modal View to create a new Event like shown below:

image

This component should use the API developed here #133 

Note: Create the components with the appropriate style. Please check the wireframes https://www.figma.com/file/9gN79UmpDPrAZk5jY4V7Ww/Study-Group-Management-System?node-id=0%3A1

### PR DESCRIPTION
**Pull request by Deepti-Dk**
# Description

Created Event Modal. Updated 'input' , 'datetime', 'modal' components.

Fixes #132 

# How to test?

Through Storybook 

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by 1620prabn — DISMISSED** [review]
good work

**Review by MaherHussain — APPROVED** [review]

File: src/server/api/routes/api-router.js [hunk 1]
```diff
@@ -45,6 +46,9 @@ router.use('/topics', TopicsRouter);
 router.use('/skills', skillsRouter);
 
 router.use('/events', eventsRouter);
+
 router.use('/groups', groupsRouter);
 
+router.use('/createEvent', createEventRouter);
```
Comment (tpetrina) [inline src/server/api/routes/api-router.js:52]: You should do this under the already existing `/events` router seen in line 48.

File: src/server/api/routes/create-event.router.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ *            max_participants:
+ *              type: integer
+ *    responses:
+ *      201:
+ *        description: createEvent created
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.post('/', (req, res) => {
+  console.log('event posted');
```
Comment (tpetrina) [inline src/server/api/routes/create-event.router.js:46]: Remove console.log statements.

File: src/server/api/routes/create-event.router.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ *              type: string
+ *            max_participants:
+ *              type: integer
+ *    responses:
+ *      201:
+ *        description: createEvent created
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.post('/', (req, res) => {
```
Comment (tpetrina) [inline src/server/api/routes/create-event.router.js:45]: Move to the eventsRouter file.

**Review by tpetrina — CHANGES_REQUESTED** [review]
This PR has way too many changes in frontend and backend. You should split it so both can move at their own pace.

File: src/server/api/routes/create-event.router.js (hunk 2 above)
Comment (Deepti-Dk) [reply src/server/api/routes/create-event.router.js:46]: > Remove console.log statements.

code embedded in events.router.js, sorted

File: src/server/api/routes/create-event.router.js (hunk 3 above)
Comment (Deepti-Dk) [reply src/server/api/routes/create-event.router.js:45]: > Move to the eventsRouter file.

Done

**Comment by Deepti-Dk** [comment]
A very messy PR, would push a new and neat version.

**Pull request closed without merging**